src/components/RequireAuth.tsx

```
@@ -0,0 +1,81 @@
+            // Check if token is expired
+            if (payload.exp && payload.exp * 1000 > Date.now()) {
+              // Instead of making an API call, use the decoded payload
+              setUser({
+                accountNo: payload.accountNo,
+                email: payload.email,
+                role: payload.role,
+                exp: payload.exp
+              })
+            } else {
+              // Token is expired
+              console.log('Token expired')
```

ESLint

[error] Unexpected console statement.

(no-console)